packages/app/pages/Registration/organizationRegistration.tsx

```diff
@@ -1,11 +1,87 @@
 /**
  * @author Connor Hibbs <hibbscm@msoe.edu>
  * @version 1.0.0
+ * @since 29 April 2018
  */
 
 import * as React from "react";
+import * as Router from "react-router";
+import * as Modal from "react-modal";
 import * as EmailValidator from "email-validator";
 import { Link } from "react-router";
+import createOrganization from "../../../ui-bindings/Organization/CreateOrganization.tsx";
+import * as Phone from "phone";
+// import allOrgsClass from "../../../ui-bindings/Organization/allOrgsClass.tsx";
+
+/**
+ * This popup appears after a successful registration and offers to
+ * take the user to the edit page for more information, or
+ * to redirect them to their created organization to see the finished page
+ */
+class Popup extends React.Component {
+
+  private state = {
+    showModal: (this.props.orgId !== undefined)
+  };
+
+  /** Custom styles for the wizard modal (CSS equiv) */
+  private wizardModalStyle = {
+    overlay : {
+      position          : "fixed",
+      top               : 0,
+      left              : 0,
+      right             : 0,
+      bottom            : 0,
+      backgroundColor   : "rgba(255, 255, 255, 0.75)"
+    },
+    content : {
+      position                   : "absolute",
+      top                        : "50%",
+      left                       : "50%",
+      width                      : "50%",
+      leftMargin                 : "auto",
+      rightMargin                : "auto",
+      transform                  : "translate(-50%, -50%)",
+      border                     : "1px solid #ccc",
+      background                 : "#fff",
+      overflow                   : "auto",
+      WebkitOverflowScrolling    : "touch",
+      borderRadius               : "4px",
+      outline                    : "none",
+      padding                    : "20px"
+    }
+  };
+
+  render() {
+    let id = this.props.orgId;
+    let url = "/projects/" + id;
+
+    return (
+      <div>
+        <Modal
+          isOpen={id}
+          style={this.wizardModalStyle}
+          onRequestClose={() => this.setState({showModal: false})}
+        >
+          <div>
+            <div style={{display: "block", textAlign: "right"}}>
+              <button style={{textAlign: "right"}}><Link to={url}>[ X ]</Link></button>
+            </div>
+
+            <h1>You're All Set!</h1>
+            <p>You have successfully registered your organization.
+              Now you can add more details like giving your organization a picture,
+              or adding some notes about what your organization does.</p>
+
+            <button><Link to={url}>No Thanks, Not Now</Link></button>
+            <button><Link to={url + "/edit"}>Take Me There!</Link></button>
```

Comment:
I will see what I can do!

Reply:
That looks like it is going to be a problem with the CSS again. They were green before, and spaced out. I don't know where the styling went but Button's should have CSS at the top level since those are "atoms" defined in our project